### PR TITLE
added rule for PBI titles

### PR DESCRIPTION
**Pull request by lukecookssw**
Rule has been checked by Adam, but would like additional eyes/feedback before we push it live.

### CODE REVIEW

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
+related:
+  - good-email-subject
+redirects: []
+
+---
+
+# Do you use meaningful PBI titles?
+
```
Comment (wicksipedia) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:18]:
```suggestion
```
Title is injected from the frontmatter

**Review by matt-goldman — APPROVED** [review]
Ship it

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Product Backlog Items (PBIs) are the cornerstone of a well-oiled project. They track features, bugs, tasks, and much more. When a developer or Product Owner is looking through the backlog, it's important that - at a glance -  they can read the titles of PBIs and have a decent understanding of them.
+
+<!--endintro-->
```
Comment (wicksipedia) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:21]: If someone is looking at the rule exerpts only, they may think there is nothing left in the rule... I think there needs to be some sort of question to prompt a person to read the rest of the rule

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- ✅ Prefix the area/form
+- ✅ Identify its urgency (e.g. 🔥)
+- ✅ Be specific (e.g. "[Area] | [behaviour]". See our rule to [order of instructions](https://www.ssw.com.au/rules/use-the-right-order-of-instructions))
+
+
+### Don't
+
+- ❌ Write a novel in the title
+- ❌ Be generic (e.g. "Fix bug in site")
+- ❌ Ignore the importance of urgent PBIs
```
Comment (wicksipedia) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:57]:
```suggestion
### Don't

- ❌ Write a novel in the title
- ❌ Be generic (e.g. "Fix bug in site")
- ❌ Ignore the importance of urgent PBIs


### Do

- ✅ Identify the bugs (e.g. "Bug" and/or 🐛)
Bugs are special case - they should have [greater visibiliy](https://www.ssw.com.au/rules/management-do-you-fix-bugs-first)
- ✅ Use emojis (see our rule on [emojis in scrum](https://www.ssw.com.au/rules/which-emojis-to-use-in-scrum))
- ✅ Prefix the area/form
- ✅ Identify its urgency (e.g. 🔥)
- ✅ Be specific (e.g. "[Area] | [behaviour]". See our rule to [order of instructions](https://www.ssw.com.au/rules/use-the-right-order-of-instructions))
```

bad examples before good ones

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: rules/do-you-use-meaningful-pbi-titles/rule.md (hunk 2 above)
Comment (lukecookssw) [reply rules/do-you-use-meaningful-pbi-titles/rule.md:18]: Good idea. Done.

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+created: 2022-12-08T07:23:54.0000000Z
+authors:
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
+related:
+  - good-email-subject
+redirects: []
+
+---
+Product Backlog Items (PBIs) are the cornerstone of a well-oiled project. They track features, bugs, tasks, and much more. When a developer or Product Owner is looking through the backlog, it's important that - at a glance -  they can read the titles of PBIs and have a decent understanding of them. So what separates a good PBI title from a bad one?
```
Comment (wicksipedia) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:16]:
```suggestion
Product Backlog Items (PBIs) are the cornerstone of a well-oiled project. They track features, bugs, tasks, and much more. When a developer or Product Owner is looking through the backlog, it's important that - at a glance -  they can read the titles of PBIs and have a decent understanding of them.

So what separates a good PBI title from a bad one?
```

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- DevOps/Infra:
+
+  ::: greybox
+  👷‍♂️ DevOps | Add ephemeral deployment slots for PRs
+  :::
+
+- Urgent tasks:
+
+  ::: greybox
+  🔥🐛👷‍♂️ SysAdmin | Northwind app inaccessible through company VPN
```
Comment (JackDevAU) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:88]: Do we use multiple emojis 👀👀or is a single 👀 preferred?

File: rules/do-you-use-meaningful-pbi-titles/rule.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+
+### Do
+
+- ✅ Identify the bugs (e.g. "Bug" and/or 🐛)
+Bugs are special case - they should have [greater visibiliy](https://www.ssw.com.au/rules/management-do-you-fix-bugs-first)
+- ✅ Use emojis (see our rule on [emojis in scrum](https://www.ssw.com.au/rules/which-emojis-to-use-in-scrum))
+- ✅ Prefix the area/form
+- ✅ Identify its urgency (e.g. 🔥)
+- ✅ Be specific (e.g. "[Area] | [behaviour]". See our rule to [order of instructions](https://www.ssw.com.au/rules/use-the-right-order-of-instructions))
```
Comment (wicksipedia) [inline rules/do-you-use-meaningful-pbi-titles/rule.md:56]:
```suggestion
- ✅ Identify its urgency (e.g. 🔥)
- ✅ Identify the bugs (e.g. "Bug" and/or 🐛)
Bugs are special case - they should have [greater visibiliy](https://www.ssw.com.au/rules/management-do-you-fix-bugs-first)
- ✅ Use emojis (see our rule on [emojis in scrum](https://www.ssw.com.au/rules/which-emojis-to-use-in-scrum))
- ✅ Prefix the area/form
- ✅ Be specific (e.g. "[Area] | [behaviour]". See our rule to [order of instructions](https://www.ssw.com.au/rules/use-the-right-order-of-instructions))
```

File: rules/do-you-use-meaningful-pbi-titles/rule.md (hunk 5 above)
Comment (wicksipedia) [reply rules/do-you-use-meaningful-pbi-titles/rule.md:88]: I think multiple are fine...
- 🔥 = urgent
- 🐛 = bug
- 👷‍♂️ = SysAdmin

**Review by wicksipedia — APPROVED** [review]
🚀

**Pull request merged**